src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,296 @@
+cross-origin isolated. With a cross-origin isolated state, the webpage will be 
+able to use privileged features including:
+
+* [`SharedArrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)
+  (required for WebAssembly Threads)
+* [`performance.measureMemory`](https://github.com/WICG/performance-measure-memory)
+* [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/)
+
+The cross-origin isolated state also prevents modifications of 
+`document.domain`. (Being able to alter `document.domain` allows 
+communication between same-sites and has been considered a loophole in the 
+same-origin policy.)
+
+{% Aside 'caution' %}
+These powerful features and the prevention of `document.domain` modification 
+are not yet enabled in Chrome as of version 83. We'll update this post as they 
+become available.
+{% endAside %}
+
+To opt in to a cross-origin isolated state, you need to send the following two 
+HTTP headers on the main document:
+
+```http
+Cross-Origin-Embedder-Policy: require-corp
+Cross-Origin-Opener-Policy: same-origin
+```
+
+These headers instruct the browser to block loading of resources or iframes 
+which haven't opted into being loaded by cross-origin documents, and prevent 
+cross-origin windows from directly interacting with your document. This also 
+means those resources being loaded cross-origin require opt-ins.
+
+You can determine whether a web page is in a cross-origin isolated state by 
+examining `self.crossOriginIsolated`. (This works on Firefox but has yet to be 
+implemented in Chrome).
+
+This article shows how to use these new headers. In a follow-up article provide 
+more background and context.
+
+{% Aside %}
+This article is aimed at those who would like to get their websites ready for 
+using `SharedArrayBuffer`, WebAssembly Threads, `performance.measureMemory`
+or the JS Self-Profiling API in a more robust manner across browser 
+platforms.
+{% endAside %}
+
+{% Aside 'key-term' %}
+This article will use many similar-sounding terminologies. To make things 
+clearer, let's enumerate them first:
+
+* [COEP: Cross Origin Embedder 
+  Policy](https://wicg.github.io/cross-origin-embedder-policy/)
+* [COOP: Cross Origin Opener 
+  Policy](https://github.com/whatwg/html/pull/5334/files)
+* [CORP: Cross Origin Resource 
+  Policy](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP))
+* [CORS: Cross Origin Resource 
+  Sharing](https://developer.mozilla.org/docs/Web/HTTP/CORS)
+* [CORB: Cross Origin Read 
+  Blocking](https://www.chromium.org/Home/chromium-security/corb-for-developers)
+{% endAside %}
+
+## Deploy COOP and COEP to make your website cross-origin isolated
+### Integrate COOP and COEP
+#### 1. Set the `Cross-Origin-Opener-Policy: same-origin` header on the top-level document
+
+By enabling COOP (Cross Origin Opener Policy) on a top-level document, windows 
```

Comment:
```suggestion
By enabling COOP on a top-level document, windows 
```

Reply:
You've already defined this, so you don't need to do it again. 

For future reference, the standard way of doing this in text is:
> Cross Origin Opener Policy (COOP)
This is the reverse of what you've done.